automation/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:15.04
+
+# Install the following utilities (required by poky)
+RUN apt-get update && apt-get install -y build-essential chrpath curl diffstat gcc-multilib gawk git-core libsdl1.2-dev texinfo unzip wget xterm cpio file && rm -rf /var/lib/apt/lists/*
+
```

Comment:
Can't the "rm -rf /var/lib/apt/lists/*" be after the last RUN command that uses apt-get? Do we have to remove it every time and then on the next RUN command to do "apt-get update" again and "rm -rf /var/lib/apt/lists/*" at the end of the RUN command?